claat/types/meta.go

```diff
@@ -64,6 +65,12 @@ type Codelab struct {
 	Steps []*Step
 }
 
+func NewCodelab() *Codelab {
+	clab := new(Codelab)
+	clab.Extra = make(map[string]string)
```

Comment:
Why `make` here?

Reply:
Without it, the map is uninitialized and assignment to it fails.

Reply:
Empty Map literal doesn't work? `map[string]string{}`

Per 

- https://stackoverflow.com/questions/16959992/creating-map-with-without-make
- https://stackoverflow.com/questions/31064688/which-is-the-nicer-way-to-initialize-a-map-in-golang

Reply:
The last convention for `make` I heard was to use it whenever possible because a single keyword is simpler and makes the author's intention more explicit than creating an empty struct and taking the address of it. I don't have a particularly strong standing either way.

Reply:
Yeah I don't see how it's explicit. `make` reminds me of C/++ of a dynamic-memory/heap allocation, and non-sized maps don't really fit that category since maps are meant to be mutable (they can't be labelled as `const`)

Feel free to resolve as you see fit.

Reply:
Switched to empty map declaration. Cannot combine this with the above line because Extra comes from the embedded Meta struct.